pb/summary/summary.proto

```diff
@@ -87,7 +140,11 @@ message DashboardTabSummary {
   // String indicating the URL for linking to a bug.
   string bug_url = 10;
 
-  reserved 11;
+  // Failure Ratios for columns of a tab
+  repeated float column_failure_ratios = 11;
```

Comment:
nit: why this shows up in your PR?

Reply:
I'm really not sure how that got there. Maybe it happened in the mess of me trying to squash my commits, but I'll revert it.

Reply:
I'm really not sure how that got there. Maybe it happened in the mess of me trying to squash my commits, but I'll revert it.